tutorials/2d/2d_lights_and_shadows.rst

```diff
@@ -0,0 +1,226 @@
+.. _doc_2d_lights_and_shadows:
+
+Working with lights and shadows in 2D
+=====================================
+
+Introduction
+------------
+
+This tutorial will explain how the 2D lighting works in the 
```

Comment:
will explain -> explains (most of the tutorial is written in present tense, so I'm recommending removing the future tense uses)